src/data/int/parity.lean

```diff
@@ -195,6 +192,18 @@ begin
   simp with parity_simps
 end
 
+@[simp, norm_cast] theorem even_coe_nat (n : ℕ) : even (n : ℤ) ↔ even n :=
+by rw_mod_cast [even_iff, nat.even_iff]
+
+@[simp, norm_cast] theorem odd_coe_nat {n : ℕ} : odd (n : ℤ) ↔ odd n :=
+by rw [odd_iff_not_even, nat.odd_iff_not_even, even_coe_nat]
```

Comment:
I see `n` is explicit in the preexisting lemma, so I suppose we should make it explicit in this one as well to match.